paper.md

```diff
@@ -0,0 +1,65 @@
+
+This paper discusses several new features that fill this gap, included in SciPy 1.15.0:
+
+- `scipy.differentiate`, a sub-package for numerical differentiation of scalar or vector-valued functions of one or more variables,
+- `scipy.integrate.tanhsinh` for quadrature of scalar or vector-valued integrands of one variable,
+- `scipy.integrate.nsum` for summation of real-valued finite or infinite series,
+- `scipy.optimize.elementwise` for finding roots and minimization of a single-input, single output function.
+
+Although the details of the algorithms are inherently distinct, these features rely on a common framework for elementwise iterative methods and share similar interfaces. For example, `scipy.differentiate.derivative` accepts the scalar function to be differentiated as a Python callable, the points at which to evaluate the derivative as an N-dimensional array, and any other parameters of the callable as N-dimensional arrays. `derivative` passes abscissae and parameters to the callable as N-dimensional arrays, adaptively approximating the derivative(s) to achieve specified tolerances. Furthermore, arrays need not be NumPy arrays: the same implementation works with several Python Array API Standard [@arrayapi] compatible arrays, such as PyTorch [@pytorch] tensors and CuPy [@cupy] arrays. These features will dramatically improve performance of end-user applications, and together, they form the backbone of SciPy's new random variable infrastructure.
+
+# Statement of need
+
+Before the release of SciPy 1.15.0, the need for these capabilities was partially met in the scientific Python ecosystem. As popular examples, Numdifftools [@numdifftools] and PyNumDiff [@PyNumDiff] provide tools for numerical differentiation, `quadpy` [@quadpy] and `scipy.integrate` offer several functions for numerical integration, `mpmath`'s [@mpmath] `nsum` function is for series summation, and `scipy.optimize` offers functions for scalar minimization and root finding. However, to the authors' knowledge, the new implementations are unique in that they offer the following advantages.
+
+- Backend-independence: all of these features are "Array API compatible" in the sense that they rely almost entirely on the Python standard libary and calls to Python Array API Standard functions/methods. Consequently, the functions accept objects other than NumPy arrays, such as PyTorch tensors and CuPy arrays, and the calculations are performed by the underlying array library.
```

Comment:
```suggestion
- Backend-independence: all of these features are "Array API compatible" in the sense that they rely almost entirely on the Python standard library and calls to Python Array API Standard functions/methods. Consequently, the functions accept objects other than NumPy arrays, such as PyTorch tensors and CuPy arrays, and the calculations are performed by the underlying array library.
```

Reply:
I should probably add a spell checker plugin to my text editor : )